### DISCUSSION TITLE
Actual PWA Support

### DISCUSSION THREAD

**Discussion by MulverineX** [jesec/flood#331] [Ideas]
Suggestion: add a basic service worker so browsers allow you to install and use Flood as a PWA.

**Comment by jesec** [comment]
There are some troubles with authentication, basic auth specifically. I also have some concerns about the stale cache. 

https://github.com/jesec/flood/projects/6#card-51205496

I would take a look at the issue.

**Reply by MulverineX** [reply]
TBH what I would do is just switch it to just cache the basic Webpack bundles; vendor, project, and styles. I think there are guides to do that. This would prevent all the headaches related to auth & outdated info.

**Comment by C84186** [comment]
Mozilla is pulling support for PWAs.
Idk about u, but my "torrent browser" is a firefox instance. 
Unless mozilla changes its mind, I'd be surprised if PWAs are really going to become all that

**Reply by MulverineX** [reply]
The far majority of web traffic is chromium even if you eliminate the mobile market. Mozilla has to choose it's battles due to it's much smaller audience, it's understandable that they don't make the time for a new effort-intensive minor standard they don't control. Due to aforementioned majority in chromium though, I don't think it's out of the question for projects to prioritize some features, even if they're "exclusive" (they're forks, and many are significantly different than base)

**Reply by C84186** [reply]
Good point, also I think I heard the other day that Mozilla is moving back towards PWA
